app/views/friend_requests/received_request.html.erb

```diff
@@ -5,28 +5,35 @@
 
     <div class="col-8">
       <div class="card">
-        <% if current_user.pending_requests_from_users.count.positive? %>
+        <% if current_user.incoming_pending_friends.count.positive? %>
           <div class="card-header d-flex flex-column">
             <span>
-              <strong>Respond to Your <%= pluralize(current_user.pending_requests_from_users.count, 'Friend Request')%> </strong>
+              <strong>Respond to Your <%= pluralize(current_user.incoming_pending_friends.count, 'Friend Request')%> </strong>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).